3cn-ecn/nantralPlatform
# change cowlocathlon to colocathlon

# Description

* Rename all "cowlocathlon" words to "colocathlon"
* Correct a type error in the frontend about roommates for the colocathlon
* New admin view to reset info about the colocathlon and to have stats too.

# Tests

1. Steps to reproduce :
    * Activate the colocathlon mode in the settings
    * Ensure that the "word" colocathlon is written everywhere instead of "cowlocathlon"
    * Go to the url colocs/admin and test the reset button
2. Expected results :
    * Did not have time to test, sorry ^

# Screenshots

Not this time, sorry ^^
